usagov/vote-gov-drupal
# Vote-841 -> links migration

**_Delete all details that do not apply to this PR!_**

## Jira ticket (required)
https://bixal-projects.atlassian.net/browse/VOTE-841

## Description (optional)
This PR will add link validator test to the project.  Right now these are place holder until content is present on site and will be updated to cover and test the correct parts of the site. 

## Deployment and testing (required, if applicable)
### Testing steps
1. pull down branch 
2. `lando start`
3. cd into `testing` folder 
4. `npm run cy:open` and check excluded, external and internal links test are passing 
5. `npm run cy:links` -> internal links test headless and `npm run cy:proofer` -> external links test headless
